### DISCUSSION TITLE
I can't create event and add googlemeet

### DISCUSSION THREAD

**Discussion by b23618** [spatie/laravel-google-calendar#286] [Q&A]
my code
`  $event = new Event;
        $event->name = $title;
        $event->description = $description;
        $event->startDateTime = Carbon::now();
        $event->endDateTime = Carbon::now()->addHour();
        $event->addMeetLink(); // optionally add a google meet link to the event
        $event->save();`

help me pleaseeeee

**Comment by sukhrobnurali** [comment]
same shit happaning with me. can't create event with google meet link with service account.

**Comment by syntx1997** [comment]
Dont forget to put: GOOGLE_CALENDAR_IMPERSONATE on .env file.
